src/__experimental__/components/select/select.stories.js

```diff
@@ -24,8 +24,10 @@ const commonKnobs = (store) => {
     errorMessage: text('errorMessage', ''),
     infoMessage: text('infoMessage', ''),
     label,
-    labelAlign: label !== '' ? select('labelAlign', OptionsHelper.alignBinary) : undefined,
-    labelInline: label !== '' ? boolean('labelInline', false) : undefined,
+    labelAlign: (
+      label.length ? select('labelAlign', OptionsHelper.alignBinary, OptionsHelper.alignBinary[0]) : undefined
```

Comment:
I don't like setting a value as `undefined` it should remain a value only used by the engine, please can you use `null` instead.

Reply:
Setting it to `null` will pass the prop down to the component and cause the warnings to appear again, setting it to `undefined` simply prevents the prop being passed. Instead I've just refactored it to remove the ternaries.